assembl/static2/routes.json

```diff
@@ -53,6 +53,7 @@
   "exportThreadMulticolumnData": "data/Discussion/${debateId}/phase2_csv_export${translation}${anonymous}",
   "exportVoteSessionData": "data/Discussion/${debateId}/widgets/${voteSessionId}/${exportRoute}",
   "exportTaxonomiesData": "data/Discussion/${debateId}/extract_csv_taxonomy",
+  "discussionPreferencesAdmin": "${slug}/administration/discussion?section=1",
```

Comment:
Once #679 is merged, section will be 2. We should probably use the SECTION_DISCUSSION_PREFERENCES constant instead of creating a new route to be sure to not have future regression.